src/constructs/ecs/__snapshots__/ecs-task.test.ts.snap

```diff
@@ -3,6 +3,107 @@
 exports[`The GuEcsTask pattern should create the correct resources with lots of config 1`] = `
 {
   "Mappings": {
+    "DefaultCrNodeVersionMap": {
```

Comment:
Do we know where these new mappings originate from? Feels unrelated to the change we're making in this PR, so it's surprising to see.